js/bibox.js

```diff
@@ -813,7 +813,14 @@ module.exports = class bibox extends Exchange {
         //         'status': 3
         //     }
         //
-        const id = this.safeString (transaction, 'id');
+        // withdraw
+        //
+        //     {
+        //         "result": 228, // withdrawal id
+        //         "cmd":"transfer/transferOut"
```

Comment:
We can also set type = 'withdraw' if it's `transferOut` and type = 'deposit' if it's `transferIn`, i think.

Reply:
Yes, sure, I will have to address the field-parsing later, just on the first bulk-update, my purpose is not to make heavy PR's for each exchange, instead in this run, I want to apply unified 'parseTransaction' routing to each `withdraw` function, then when I'm done with them across all missing exchanges, then i will have to start 'round 2' , which includes individual check of exchange responses and parsing their fields correctly.
but again, in current scope, to make the merging simple and non-breaking, i am just migrating the current code through `parseTransaction` to have a light PR's (to easily merge), after that, the slow process of 'round 2' will start.